tests/helper/config_test.cpp

```diff
@@ -9,9 +9,8 @@
 namespace dds {
 
 namespace {
-template<typename T, size_t size>
-size_t vsize(T(&)[size]){return size;}
-}
+template <typename T, size_t size> size_t vsize(T (&)[size]) { return size; }
```

Comment:
Could be `constexpr`.

Reply:
This is just here because format_fix modified it :-\